### PR TITLE
3143 orange bar setup

### PR DESCRIPTION
**Pull request by dhochbaum-dcp**
#### Tasks/Bug Numbers
 - Fixes [AB#1234](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/1234)

 - Related [AB#1235](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/1235)
 - Reverses [AB#1222](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/1222)

### CODE REVIEW

File: app/components/orange-bar-custom-map-study.js [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+import { computed } from '@ember/object';
+import { task } from 'ember-concurrency';
+import numeral from 'numeral';
+import fetch from 'fetch';
+import Environment from '../config/environment';
+import choroplethConfigs from '../choropleth-config';
+
+const { SupportServiceHost } = Environment;
+
+export default Component.extend({
```
Comment (allthesignals) [inline app/components/orange-bar-custom-map-study.js:13]: I'm assuming this file is the combination of other code elsewhere

File: app/styles/base/_typography.scss [hunk 2]
```diff
@@ -516,6 +516,9 @@ tr {
 // --------------------------------------------------
 .draw-tools--container {
   position: relative;
+  width: 150px;
+  height: 33.21px;
+  left: 25px;
```
Comment (allthesignals) [inline app/styles/base/_typography.scss:521]: Next step is to start the mobile compatibility. Not MVP but let's keep those problems in mind. Pixel dimensions are red flags

File: app/styles/layouts/_l-default.scss [hunk 3]
```diff
@@ -159,10 +159,12 @@ body {
   @include breakpoint(large) {
     position: absolute;
     top: 10px;
-    left: 50px;
+    // left: 50px;
+    right: 20px;
     z-index: 4;
     padding-bottom: 0;
-    margin-top: rem-calc(92);
+    // margin-top: rem-calc(92);
```
Comment (allthesignals) [inline app/styles/layouts/_l-default.scss:166]: I'd just remove this sort of thing than keep it commented out

File: app/templates/components/orange-bar-custom-map-study.hbs [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                {{fa-icon icon='dot-circle' prefix='far'}}
+                <span class="hide-for-medium">Draw</span> Radius
+              </a>
+            </div>      
+          </div>
+        </li>
+      </ul>
+    </li>
+    <li class="search">
+      {{!-- Map-Search should go here, but putting it in was causing page not to load --}}
```
Comment (allthesignals) [inline app/templates/components/orange-bar-custom-map-study.hbs:27]: Is this comment still true?

File: app/templates/components/orange-bar-data-explorer.hbs [hunk 5]
```diff
@@ -0,0 +1 @@
+<li>Orange bar for the data explorer page</li>
```
Comment (allthesignals) [inline app/templates/components/orange-bar-data-explorer.hbs:1]: image

I like to check for these in general.

File: app/templates/index.hbs [hunk 6]
```diff
@@ -1,4 +1,8 @@
-<ul class="menu align-center menu--selector">
+{{#orange-bar}}
+  {{#orange-bar-custom-map-study}}{{/orange-bar-custom-map-study}}
+{{/orange-bar}}
```
Comment (allthesignals) [inline app/templates/index.hbs:3]: Good. 

You can also leverage folders for component organization. For example:

orange-bar/custom-map-study.hbs would be:

```handlebars
{{#orange-bar}}
   {{orange-bar/custom-map-study}}
{{/orange-bar}}
```

Note that components without blocks can be invoked without `#`.

**Review by allthesignals — COMMENTED** [review]
This is pretty good, but I would have split out the "custom map study" component into even more components:

```
┌────────────┐                       
│ orange-bar/ ───┐                   
└────────────┘   │                   
                 │   ┌──────────────┐
                 ├──▶│ /more-options.hbs 
                 │   └──────────────┘
                 │                   
                 │   ┌──────────────┐
                 ├──▶│    /search.hbs    
                 │   └──────────────┘
                 │                   
                 │   ┌──────────────┐
                 └──▶/summary-level.hbs 
                     └──────────────┘
```

The question to ask when splitting up a component is "does this part of the component share any state with its siblings?" 

I'm going to fix up this PR because I know you're doing other things. Just wanted to share some thoughts.

File: app/components/orange-bar-custom-map-study.js (hunk 1 above)
Comment (dhochbaum-dcp) [reply app/components/orange-bar-custom-map-study.js:13]: Completely taken from map-utility-box.js

File: app/templates/components/orange-bar-custom-map-study.hbs (hunk 4 above)
Comment (dhochbaum-dcp) [reply app/templates/components/orange-bar-custom-map-study.hbs:27]: No, it works now.

File: app/styles/layouts/_l-default.scss (hunk 3 above)
Comment (dhochbaum-dcp) [reply app/styles/layouts/_l-default.scss:166]: Clearly I didn't do a great job of scrubbing out all of my unnecessary comments...

**Pull request merged**